subprojects/docs/src/samples/userguide/buildlifecycle/buildProjectEvaluateEvents/settings.gradle

```diff
@@ -1,3 +1,5 @@
+rootProject.name = 'buildProjectEvaluateEvents'
```

Comment:
why is this needed?

Reply:
This is similar to above, to allow output verification to match the project name.